core/iwasm/common/wasm_runtime_common.h

```diff
@@ -167,6 +167,18 @@ wasm_runtime_call_wasm(WASMExecEnv *exec_env,
                        WASMFunctionInstanceCommon *function,
                        uint32 argc, uint32 argv[]);
 
+bool
+wasm_runtime_call_wasm_a(WASMExecEnv *exec_env,
+                       WASMFunctionInstanceCommon *function,
```

Comment:
Indent: had better align to first argument of last line, if current line's length is not so long.